### PR TITLE
Fixes #11427 - puppetclass edit text under optional input validator i…

### PR DESCRIPTION
**Pull request by xprazak2**
…s now alert-info

### CODE REVIEW

File: app/views/lookup_keys/_fields.html.erb [hunk 1]
```diff
@@ -31,7 +31,7 @@
   <fieldset>
     <%= collapsing_legend _("Optional input validator"), "#optional_input_validators_#{f.object.id}", "collapsed" %>
     <div id="optional_input_validators_<%= f.object.id %>" class="collapse out">
-      <p class="help-block">
+      <p class="alert alert-info">
```
Comment (orrabin) [inline app/views/lookup_keys/_fields.html.erb]: @xprazak2 you can use the alert helper: https://github.com/theforeman/foreman/blob/develop/app/helpers/layout_helper.rb#L36
If you want to keep it looking the same:
`<%= alert :class => 'alert-info', :header => '', :text => _('Note that if you use ERB as a value of parameter, value will be validated during ENC evaluation. If value is invalid, ENC evaluation will fail.') %>` 
Or move the `Note` to the header to make it more noticeable.

**Comment by xprazak2** [comment]
[test]

**Comment by dLobatog** [comment]
[test]

File: app/views/lookup_keys/_fields.html.erb [hunk 2]
```diff
@@ -31,9 +31,8 @@
   <fieldset>
     <%= collapsing_legend _("Optional input validator"), "#optional_input_validators_#{f.object.id}", "collapsed" %>
     <div id="optional_input_validators_<%= f.object.id %>" class="collapse out">
-      <p class="help-block">
-        <%= _('Note that if you use ERB as a value of parameter, value will be validated during ENC evaluation. If value is invalid, ENC evaluation will fail.') %>
-      </p>
+      <%= alert :class => 'alert-info', :header => 'Note', :text => _('If you use ERB as a value of parameter, value will be validated during ENC evaluation. If value is invalid, ENC evaluation will fail.') %>
```
Comment (dLobatog) [inline app/views/lookup_keys/_fields.html.erb]: Can you change this to `If ERB is used in a parameter value, the validation of the value will happen during the ENC request. If the value is invalid, the ENC request will fail.` as @domcleal mentioned on #theforeman-dev? Maybe remove the Note header, or if you want to leave it there, extract the string with _()

**Comment by dLobatog** [comment]
Merged as 06c6f2d2eadb149e974d379dad17761ffd5c3c4c, thanks @xprazak2!

**Pull request closed without merging**